test/integration/user-management-api/buyers_account_plan_change_test.rb

```diff
@@ -1,63 +1,62 @@
+# frozen_string_literal: true
+
 require 'test_helper'
 
 class EnterpriseApiBuyersAccountPlanChangeTest < ActionDispatch::IntegrationTest
   def setup
-    @provider = FactoryBot.create :provider_account, :domain => 'provider.example.com'
-    @buyer = FactoryBot.create(:buyer_account, :provider_account => @provider)
+    @provider = FactoryBot.create(:provider_account, domain: 'provider.example.com')
+    @buyer = FactoryBot.create(:buyer_account, provider_account: @provider)
     @buyer.buy! @provider.default_account_plan
     @buyer.reload
 
-    @published_account_plan = FactoryBot.create :account_plan, :issuer => @provider
+    @published_account_plan = FactoryBot.create(:account_plan, issuer: @provider)
     @published_account_plan.publish!
 
-    @hidden_account_plan = FactoryBot.create :account_plan, :issuer => @provider
+    @hidden_account_plan = FactoryBot.create(:account_plan, issuer: @provider)
 
     host! @provider.admin_domain
   end
 
   test 'successful change account plan' do
     put change_plan_admin_api_account_path(@buyer,
-                                                :provider_key => @provider.api_key,
-                                                "plan_id" => @published_account_plan.id,
-                                                :format => :xml)
-
+                                           provider_key: @provider.api_key,
+                                           plan_id: @published_account_plan.id,
+                                           format: :xml)
     assert_response :success
 
-    assert @buyer.bought_account_plan == @published_account_plan
+    assert_equal @buyer.bought_account_plan, @published_account_plan
```

Comment:
See https://github.com/3scale/porta/pull/2646/files#r752312663

Reply:
Right, can we let this one pass though?